content/ko/continuous-delivery.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 지속적인 전달 (Continuous Delivery)
+status: Completed
+category: 개념
+tags: ["방법론", "애플리케이션", ""]
+---
+
+## 개념
+
+CD로 축약되는 지속적인 전달은,
+코드 변경사항이 승인 환경에서 자동으로 배포되는 일련의 작업이다.
+(또는, 프로덕션(production)에서 지속적인 배포의 경우)
+CD는 배포 전에 소프트웨어를 적절하게 테스트하고
+필요한 경우 변경사항을 롤백(rollback)하는 방법을 제공하는 절차가 포함된다.
+지속적인 통합 (CI)은 지속적인 전달을 향하는 첫 번째 단계이다.
+(즉, 변경사항을 테스트 및 배포하기 전에 완전히 병합해야 한다).
+
+## 다루는 문제
+
+[신뢰할 수 있는](/reliability/)업데이트 배포는 대규모에서 문제가 된다.
+이상적으로, 최종 사용자에게 더 나은 가치를 제공하기 위해 더 자주 배포해야 한다.
+그러나, 이를 수동으로 수행하면 모든 변경에 대해 높은 트랜잭션(transaction) 비용이 발생한다.
+역사적으로, 이러한 비용을 피하기 위해, 조직은 다소 적게 릴리스를 하고,
+한번 더 많은 변경 사항을 배포하고 문제가 발생할 위험이 증가했다.
```

Comment:
[원문]
> Historically, to avoid these costs, organizations have released less frequently, deploying more changes at once and increasing the risk that something goes wrong.

```suggestion
역사적으로, 이러한 비용을 피하기 위해, 조직들은 릴리스를 덜 자주 해 왔는데,
이로 인해 한번에 더 많은 변경사항을 배포하게 되어 문제가 발생할 위험이 증가했다.
```